code-challenges.test.js

```diff
@@ -65,7 +111,23 @@ var hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
 
 
-
-
-
 // b) Create the function that makes the test pass.
+
+//create a function called fullHouse
+//pass in array
+const fullHouse = (array) => {
+//set counter variable
+    let counter = 0
+//use map method
+    let m = array.map((value,index)  => {
+//conditional if value equals value at index 1, and want to compare if value 1 equals value 2 and so on
+    if(value === value[1]){
+        return
+    }
+    })
+//return counter
+    return 
```

Comment:
return _____?

Reply:
same was confused with this question and how to approach it